nncf/torch/model_graph_manager.py

```diff
@@ -337,3 +337,21 @@ def get_fake_quantizer(
             if isinstance(module, BaseQuantizer):
                 return module
     return None
+
+
+def get_target_dim_for_weight_compression(metatype: om.PTOperatorMetatype, input_port_id: int) -> int:
```

Comment:
Can you migrate to OV layout approach? @daniil-lyakhov, what do you think about it?

Reply:
It would help us in case we will decide to implement channel alignment algorithm in future for example, or other algorithms which requires dimensions other than `output_channels`, but for now only the `output_channels` are required by algorithms, so it is ok to use the function like `get_target_dim_for_weight_compression`